examples/terraform-submodules/gke/.gitignore

```diff
@@ -0,0 +1 @@
+*.tfvars
```

Comment:
I'm not sure it's worth checking this file in. The examples don't show using tfvars files; they pass all variables on the command line.